.claude/skills/roughcut/templates/apply_recipe.py

```
@@ -0,0 +1,345 @@
+            ok = bool(item.SetClipColor(clip["color_tag"]))
+            if ok:
+                self.counts["color_tags"][0] += 1
+            else:
+                self.warnings.append(f"clip {idx}: SetClipColor({clip['color_tag']!r}) returned False")
+        except Exception as e:
+            self.warnings.append(f"clip {idx}: SetClipColor raised {type(e).__name__}: {e}")
+
+    def _apply_markers(self, item, clip, idx):
+        for marker in clip.get("markers", []):
+            self.counts["markers"][1] += 1
+            frame = int(round(marker["at"] * self.frame_rate))
```

Ruff (0.15.12)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)